var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -50,7 +52,7 @@ def url_for_version(self, version):
     depends_on('py-pybind11@2.6:', type='build', when='@4.1.0:')
 
     for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0', '3.10.0', '4.0.0', '4.1.0',
-                '4.2.0', '4.3.0', '4.3.1']:
+                '4.2.0', '4.3.0', '4.3.1', '4.5.0']:
```

Comment:
Just above here there's a python requirement mismatch with the new patch.  It has cmake looking only for 3.5+, the package is looking for 3+.

Reply:
@trws, i did not fully understood the comment. but the reason the patch to the migraphx was added was that there was issue #24429 reported but  i could not reproduce it in spack environment.

Reply:
The patch means only python 3.5 and higher will work, but the package is only requesting 3 or higher.

Reply:
i tried with depends_on('python@3.5:', type='build') and it builds successfully. let me know if this change will do ?

Reply:
Yup, that works for me.